src/Uno.UI.Tests/Windows_Globalization/Given_Rounder.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.Globalization.NumberFormatting;
+
+namespace Uno.UI.Tests.Windows_Globalization
+{
+	[TestClass]
+    public class Given_Rounder
```

Comment:
Nice PR !

Those tests are a good candidate to be running as runtime tests as well. You can reference globalization the two test files in here:

https://github.com/unoplatform/uno/blob/master/src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.Wasm.csproj#L35-L48

and the other projects as well. We probably should extract those references into an additional `.props` file so we don't need to modify all `Uno.UI.RuntimeTests.*.csproj` files.

The interest here is to be able to run those same tests on windows to validate that they are behaving the same as our implementation.

Reply:
Thanks for your suggestion. I added references. Did I do it correctly?